plugins/c9.ide.ui/lib_apf.js

```diff
@@ -3726,7 +3726,7 @@ apf.setNodeValue = function(xmlNode, nodeValue) {
     else {
         var oldValue = xmlNode.nodeValue;
         xmlNode.nodeValue = nodeValue === undefined || nodeValue === null ||
-            nodeValue == NaN ? "" : String(nodeValue);
+            isNaN(nodeValue) ? "" : String(nodeValue);
```

Comment:
Oh dear, I had forgotten about that particular footgun... thanks for spotting it!